### DISCUSSION TITLE
Login route

### DISCUSSION THREAD

**Discussion by amosmos** [osiset/laravel-shopify#714] [Q&A]
Hi,

I was wondering why was the login route, which allows stores to install the app directly from the app, removed from the package.

Is it possible to install the app on a store by adding a "shop" query parameter to the home route? Is there any other way?

Thanks!
Amos

**Comment by gnikyt** [comment]
Too many reported issues with the login route and/or stores using it wrongly. You should be able to send a form with method="get" to "/authenticate?shop=(shop domain)" and it should work for you.

**Reply by amosmos** [reply]
Thanks!
